godot-core/src/builtin/string/gstring.rs

```diff
@@ -225,6 +225,21 @@ impl From<&str> for GString {
     }
 }
 
+impl From<&[char]> for GString {
+    fn from(chars: &[char]) -> Self {
+        unsafe {
```

Comment:
Since this isn't completely obvious, you should maybe add a `// SAFETY` line saying something about UTF-32.